webapp/components/NWBListViewer.js

```diff
@@ -144,13 +145,13 @@ export default class NWBListViewer extends Component {
       </div>
       
       <div className='list-summary'>
+        <a title="Plot all timeseries" onClick={this.plotAllInstances}>
+          <RemoveRedEyeIcon />
```

Comment:
There is another component with the remove eye icon, maybe not used anymore?

Reply:
Made that parameterized, so not changing here